cmd/juju/storage/poolcreate.go

```diff
@@ -72,10 +72,10 @@ func (c *poolCreateCommand) Init(args []string) (err error) {
 		return err
 	}
 
+	c.attrs = make(map[string]interface{})
```

Comment:
Do we need a test that an 'empty' c.attrs is handled correctly? (i.e. no assumptions that c.attrs always contains something)

Reply:
In this layer, it is tested by https://github.com/juju/juju/blob/develop/cmd/juju/storage/poolcreate_test.go#L43.

Everything higher up depends on the storage provider. Each different type have their own validation. Types that require attributes will fail without them. Types that do not require attributes, will not :D

Reply:
Sweet, sounds good to me.